eZ/Publish/Core/REST/Server/Input/Parser/Criterion/DateMetadata.php

```diff
@@ -28,6 +31,30 @@ class DateMetadata extends BaseParser
      */
     public function parse(array $data, ParsingDispatcher $parsingDispatcher)
     {
-        throw new \Exception('@todo implement');
+        if (!array_key_exists('DateMetadataCriterion', $data)) {
+            throw new Parser('Invalid <DateMetadataCriterion> format');
+        }
+
+        if (!isset($data['DateMetadataCriterion']['Target'])) {
+            throw new Parser('Invalid <Target> format');
+        }
+        $target = $data['DateMetadataCriterion']['Target'];
+
+        if (!isset($data['DateMetadataCriterion']['Operator'])) {
+            $data['DateMetadataCriterion']['Operator'] = 'EQ';
+        } elseif (!in_array($data['DateMetadataCriterion']['Operator'], array_keys(Operator::getAvaliableOperators()))) {
```

Comment:
You can move the method to this class as `private function getAvaliableDateMetadateRESTOperators()`, and it can return plain array with the available operators.